docs/user/alerting/rule-types.asciidoc

```diff
@@ -74,6 +74,11 @@ beta:[] {ml-docs}/ml-configuring-alerts.html[{ml-cap} rules] run scheduled check
 
 Security rules detect suspicious source events with pre-built or custom rules and create alerts when a rule’s conditions are met. For more information, refer to {security-guide}/prebuilt-rules.html[Security rules].
 
+[NOTE]
+==============================================
+Alerts associated with Security rules are not visible in the stack management view.
```

Comment:
We have some variables for the Kibana app names in https://github.com/elastic/docs/blob/master/shared/attributes.asciidoc:
```suggestion
Alerts associated with security rules are not visible in *{stack-manage-app} > {rules-ui}*.
```